client/src/components/App/App.js

```diff
@@ -16,6 +17,7 @@ const App = () => (
     <BrowserRouter>
         <Switch>
             <Route exact path="/" component={Home} />
+            <Route exact path="/login" component={Login} />
```

Comment:
Remove exact, we might have to have some hash routing on this page and use URL options as well